src/pages/Victims.scss

```diff
@@ -4,6 +4,67 @@
   .wrapper {
     width: 90%;
 
+    .searchSelect{
```

Comment:
Please fix your spacing in this file and Victims.js. Spacing should be consistent.
Add a space between the selector and {. Add a space after colons. Add a tab/2 spaces at each level. Don't add extra spaces after } like on line 60 and 62 through 65. Closing braces should line up vertically with first character of the selector.
Look at the other code to see how these things should look.